datadog/__init__.py

```diff
@@ -128,8 +138,11 @@ def initialize(
     if statsd_constant_tags:
         statsd.constant_tags += statsd_constant_tags
 
+    if statsd_disable_aggregating:
+        statsd.disable_aggregation()
+    else:
+        statsd.enable_aggregation(statsd_aggregation_flush_interval)
```

Comment:
Do you need to make sure buffering is disabled here?

Reply:
I think it should already be handled [here](https://github.com/DataDog/datadogpy/blob/add-client-side-aggregation/datadog/dogstatsd/base.py#L549) since we wont have a flush thread for buffering if aggregation is enabled